Apollo.Dockerfile

```
@@ -0,0 +1,29 @@
+FROM node:23-alpine3.20
+
+RUN npm install -g pnpm@10.6.5 @types/node typescript
+
+WORKDIR /app
+
+COPY package.json pnpm-lock.yaml checkNodeVersion.js ./
+COPY docker/apollo-server docker/apollo-server
+
+RUN pnpm install --frozen-lockfile
+
+RUN pnpm exec tsc \
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)